prometheus_client/metrics.py

```diff
@@ -553,8 +553,8 @@ def __init__(self,
         )
         self._kwargs['buckets'] = buckets
 
-    def _prepare_buckets(self, buckets):
-        buckets = [float(b) for b in buckets]
+    def _prepare_buckets(self, source_buckets: Sequence[Union[float, int, str]]) -> None:
```

Comment:
I was to rename this param, because after the fix `buckets = [float(b) for b in source_buckets]` has different type.

Reply:
Makes sense, it is an internal function so should be fine.